### ISSUE TITLE
GMail rejects autoconfig from "insecure" app (owncloud mail)

### ISSUE THREAD

**Issue by chrsch** [owncloud/mail#1213]
I tried to use the autoconfig to setup my gmail account. I use v 0.2.1 of the mail app on a ownCloud 8.2.2 release.

Within the mail app the loading spinner will turn forever. After a while I get an email from google stating the login from an insecure app has been blocked. 

Reproduce: Just get a fresh oencloud & mail setup, open mail app, just enter name, email and password and go for it.

**Comment by ChristophWurst** [comment]
I'm moving this to 0.3.1, so we can get 0.3 out. There are high chances it is fixed with the next release, at least I have never had problems with my GMail account.

**Comment by ErikPel** [comment]
You have to enable use of less secure apps from https://www.google.com/settings/security/lesssecureapps
probably should have mention about that in the readme if it's not fixed.

What I googled owncloud mail is considered less secure app because it doesn't use oauth 2

**Comment by ChristophWurst** [comment]
> probably should have mention about that in the readme if it's not fixed.

https://github.com/owncloud/mail#gmail ;-)

**Comment by chrsch** [comment]
Or even better this advice is displayed if the user tries to configure gmail... readme files are not so popular for ordinary users... ;-)

**Comment by ChristophWurst** [comment]
> Or even better this advice is displayed if the user tries to configure gmail... readme files are not so popular for ordinary users... ;-)

Sure, that would be great.

Is this issue resolved now?

**Comment by ErikPel** [comment]
@ChristophWurst 
The "Display unlock captcha" isn't the same thing as allowing your google account to use less secure apps.

**Comment by ChristophWurst** [comment]
> @ChristophWurst
> The "Display unlock captcha" isn't the same thing as allowing your google account to use less secure apps.

Yes, you're right. Then we should add that to the readme. Any volunteers? :-)
